packages/dcos-integration-test/extra/test_dcos_checks.py

```diff
@@ -0,0 +1,64 @@
+import uuid
+
+import pytest
+
+
+def test_dcos_checks_detect_ip(dcos_api_session):
+    """
+    Exercise dcos-check to make sure it runs successfully with valid
+    --detectp-ip option.
+    """
+    cmd = [
+        "/opt/mesosphere/bin/dcos-checks",
+        "--role",
+        "agent",
+        "components",
+        "--port",
+        "61001",
+        "--detect-ip",
+        "/opt/mesosphere/bin/detect_ip",
+        "--exclude=dcos-checks-poststart.service,dcos-checks-poststart.timer"
+    ]
+    test_uuid = uuid.uuid4().hex
+    job = {
+        'id': 'test-dcos-checks-detect-ip-' + test_uuid,
+        'run': {
+            'cpus': .1,
+            'mem': 128,
+            'disk': 0,
+            'cmd': ' '.join(cmd)
+        }
+    }
+
+    dcos_api_session.metronome_one_off(job)
```

Comment:
Should there be a clean-up after the one_off job? Not required if our test library code not provide that facility. We can add it later.

Reply:
metronome jobs are one shot jobs and are removed upon completion